app/assets/stylesheets/application.scss

```diff
@@ -1,44 +1,68 @@
 @import "bootstrap";
 
+$orange: #f2580b;
+@mixin orange_background {
+  background-color: $orange;
+  border-color: $orange;
+}
+body {
+  background-color: #eee;
+  font-family: 'Roboto', sans-serif;
+}
+
+.navbar-header {
+  border-bottom: 1px solid gray;
+}
+
 .container {
   margin: 48px auto;
   width: 960px;
-}
 
-.container >  h1 {
-  float: left;
-  margin: 0;
+  h1 {
+    float: left;
+    margin: 0;
+    a {
+      color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.